index.js

```diff
@@ -21,3 +97,13 @@ const allWagesFor = function () {
     return payable
 }
 
+function findEmployeeByFirstName(employeeList, firstName){
+  return employeeList.find((employee) => employee.firstName === firstName)
+}
+
+function calculatePayroll(employeeRecords){
+  let payroll = 0
+  employeeRecords.map(record => payroll += allWagesFor.call(record))
+  
+  return payroll
```

Comment:
You should either use `reduce`, or at least not be using `map`, but `forEach` instead since you're not collecting the result.